src/components/RepositorySearchCombobox.tsx

```diff
@@ -17,7 +17,7 @@ import { useCombobox } from 'downshift'
 import debounce from 'lodash/debounce'
 import * as React from 'react'
 import { useEffect, useMemo, useState } from 'react'
-import { FaArrowDown, FaArrowUp } from 'react-icons/fa'
+import { FaSearch, FaArrowUp } from 'react-icons/fa'
```

Comment:
This is the closest match to the Figma design that I could find in the current icon set. These icons should be replaced with the Heroicon icons sometime later.

At least now we use a logical icon for the default state. 😉 

I've kept the up arrow when you slide the drawer back to the top.

Reply:
Sorry but I have to disagree finally with this icon change.

This button won't search anything when pressed, but just open the select dropdown. This is even reflected in the component accessibility:

![element attribute showing it's just a toggling button for the select](https://user-images.githubusercontent.com/2677072/126895200-c75c0fa1-3d06-4986-bbf5-51663165c39a.png)

So the search icon doesn't represent its behavior properly. We need to discard this change, but we can probably include the button as part of the input itself. I didn't do this originally to make clear to the user they can type within the input, but that field is just a combo box so we can put the toggle button within it on the right side.

What do you think?

Reply:
> So the search icon doesn't represent its behavior properly. We need to discard this change,

I've discarded this change with a `git revert`. 😉 

>  but we can probably include the button as part of the input itself. I didn't do this originally to make clear to the user they can type within the input, but that field is just a combo box so we can put the toggle button within it on the right side.
> 
> What do you think?

The Figma design has a button with a search icon, that's next to the text input field.
I don't know if the Figma design is what you actually want to build?

Reply:
> The Figma design has a button with a search icon, that's next to the text input field.
I don't know if the Figma design is what you actually want to build?

The other way around: update the design to fit this behavior. I didn't think about this was a combo box with autocomplete, not a pure search. My bad 😓

Reply:
> The other way around: update the design to fit this behavior. I didn't think about this was a combo box with autocomplete, not a pure search. My bad 😓

I was wrong as well, so don't feel too bad... 😉 

We'll discuss the proper design on Figma together, as that is more visual than GitHub. 😄